unimol/unimol/utils/docking.py

```diff
@@ -61,13 +61,21 @@ def result_log(dir_path):
         default="./protein_ligand_binding_pose_prediction",
         help="Location of the docking output path",
     )
+    parser.add_argument(
+        "--optimization-model",
+        type=str,
+        default="coordinate",
```

Comment:
Maybe change to conformer? use ligand internal trosions is more reliable by default.